guides/common/assembly_installing-satellite-server-connected.adoc

```diff
@@ -0,0 +1,86 @@
+
+ifeval::["{build}" == "satellite"]
+When you install {ProjectServer} from a connected network, you can obtain packages and receive updates directly from the Red Hat Content Delivery Network.
+
+Use the following procedures to install {ProjectServer}, perform the initial configuration, and import subscription manifests.
+
+ifeval::["{build}" != "foreman-deb"]
+Before you continue, consider which manifests are relevant for your environment. For more information on manifests, see link:{BaseURL}content_management_guide/managing_subscriptions/[Managing Subscriptions] in the _Content Management Guide_.
+endif::[]
+
+
+NOTE: You cannot register {ProjectServer} to itself.
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+Use the following procedures to install {ProjectServer} and perform the initial configuration.
+endif::[]
+
+ifeval::["{build}" == "foreman"]
+On CentOS, you can install {Project} with or without the Katello plug-in. If you are a new user, consider installing {Project} with the Katello plug-in.
+endif::[]
+
+ifeval::["{build}" == "foreman-deb"]
+On Debian or Ubuntu operating systems, installations with the Katello plug-in are not supported. Ignore any references to the Katello plug-in.
+endif::[]
+
+Note that the {ProjectX} installation script is based on Puppet, which means that if you run the installation script more than once, it might overwrite any manual configuration changes.
+⁠
+To avoid this and determine which future changes apply, use the `--noop` argument when you run the installation script. This argument ensures that no actual changes are made. Potential changes are written to `/var/log/foreman-installer.log`.
+
+Files are always backed up and so you can revert any unwanted changes. For example, in the foreman-installer logs, you can see an entry similar to the following about Filebucket:
+
+[options="wrap"]
+----
+/Stage[main]/Dhcp/File[/etc/dhcp/dhcpd.conf]: Filebucketed /etc/dhcp/dhcpd.conf to puppet with sum 622d9820b8e764ab124367c68f5fa3a1
+----
+
+You can restore the previous file as follows:
+
+[options="wrap"]
+----
+# puppet filebucket -l \
+restore /etc/dhcp/dhcpd.conf 622d9820b8e764ab124367c68f5fa3a1
+----
+
+ifeval::["{build}" == "satellite"]
+include::modules/proc_registering-to-red-hat-subscription-management.adoc[leveloffset=+1]
+
+include::modules/proc_attaching-satellite-infrastructure-subscription.adoc[leveloffset=+1]
+endif::[]
+
+ifeval::["{build}" != "foreman-deb"]
+include::modules/proc_configuring-repositories.adoc[leveloffset=+1]
+endif::[]
+
+ifeval::["{build}" == "foreman-deb"]
+include::modules/proc_configuring-repositories-deb.adoc[leveloffset=+1]
+endif::[]
+
+include::modules/proc_installing-the-satellite-server-packages.adoc[leveloffset=+1]
+
+include::modules/proc_synchronizing-the-system-clock-with-chronyd.adoc[leveloffset=+1]
+
+include::modules/proc_installing-the-sos-package.adoc[leveloffset=+1]
+
+[id="configuring-server_{context}"]
+== Configuring {ProjectServer}
```

Comment:
Why is this section not a separate assembly?

Reply:
Because it isn't one. An assembly is meant to be a start-to-finish user story, and although we do stretch that definition a bit in some of our earlier assemblies, in this case it's well outside that definition as it's two different options for doing the same thing.

Reply:
Got it, thank you